Editor/Interaction/SnapZoneEditor.cs

```diff
@@ -7,28 +7,29 @@ namespace Innoactive.CreatorEditor.XRInteraction
     /// <summary>
     /// Drawer class for <see cref="SnapZone"/>.
     /// </summary>
-    [CustomEditor(typeof(SnapZone))]
+    [CustomEditor(typeof(SnapZone)), CanEditMultipleObjects]
```

Comment:
Older XRIT versions did not support it, not sure because they forgot, they didn't want people to do it, or there was a logical reason but in the end, we just stick to their practices, since now they allow it natively, I guess it is safe for us too.